crates/subspace-node/src/chain_spec.rs

```diff
@@ -78,7 +144,16 @@ pub fn testnet_config() -> Result<ChainSpec, String> {
         // Protocol ID
         Some("subspace"),
         // Properties
-        None,
+        Some(Properties::from_iter([
```

Comment:
Normally we would also expose the address prefix using `ss58Format`. Considering we haven't claimed a prefix, we could also do it when `SS58Prefix` gets updated.